src/WorldIDRouterImplV1.sol

```diff
@@ -0,0 +1,293 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {WorldIDImpl} from "./abstract/WorldIDImpl.sol";
+
+/// @title WorldID Router Implementation Version 1
+/// @author Worldcoin
+/// @notice A router component that can dispatch group numbers to the correct identity manager
+///         implementation.
+/// @dev This is the implementation delegated to by a proxy.
+contract WorldIDRouterImplV1 is WorldIDImpl {
```

Comment:
I wasn't really following the design discussions for this, so question: why does this only expose a getter and not a `verifyProof` version that takes a group id and delegates to the correct contract? Sounds like much better UX (at a slightly higher cost).

Reply:
Feel free to ignore if this was already discussed with the rest of the team.